assets/src/js/cookies.js

```diff
@@ -95,7 +95,7 @@ export const setupCookies = () => {
         ad_storage: 'granted',
         ad_user_data: 'granted',
         ad_personalization: 'granted',
-        ...ENABLE_ANALYTICAL_COOKIES && {analytics_storage: 'granted'},
```

Comment:
Same here, 
![image](https://github.com/user-attachments/assets/0ae137c1-e195-48dd-9227-2ec2553390b0)
The analytics _storage set granted by default will disrespect the above admin setting.

Reply:
See comment for line 13, it's the same situation. Disabling the setting for analytics cookies in the cookies box does not mean they are not allowed to use. Those two things are decoupled now